libraries/AP_TECS/AP_TECS.h

```diff
@@ -134,6 +134,12 @@ class AP_TECS {
         _pitch_max_limit = pitch_limit;
     }
 
+    // set minimum throttle override, 0-1 range
+    // it is applicable for one control cycle only
+    void set_min_throttle(const float thr) {
```

Comment:
maybe set_throttle_range(), and have min and max, so we can do both min and max throttle in one API
note that min can be -ve if we are allowing for reverse throttle, so need to be careful.
we could pass a NaN for the min to mean no lower limit? or maybe -1 meaning no lower limit?